### DISCUSSION TITLE
Get Game Patch

### DISCUSSION THREAD

**Discussion by RaphaelHippe** [saul/demofile#229] [Questions] [answered]
Hey, 

I was wondering if there was a way to get the CSGO game patch version of a given HLTV demofile. 

I looked for that in the demoHeader and parseHeader of your parser, the L33T/CSGO-Reflection repository for the csgo entity classes as well as CSGO and CS Source events but did not find anything.

I basically want to know on which patch (http://liquipedia.net/counterstrike/Patches) the game of the demofile was played on. 

thanks and cheers, 

Raphael Hippe

**Comment by saul** [comment, marked as answer]
Hi Raphael

Unfortunately the game version isn’t networked - the closest thing you’ll find is the networkProtocol field in the demo header. This is incremented occasionally but I don’t think there’s any rhyme or reason to it.

**Comment by RaphaelHippe** [comment]
Hi Saul, 

okay, I will have to get that information on another way than. Thanks for the response =) Keep up the good work =) 

Cheers
